test/integration/smoke/test_vm_life_cycle.py

```diff
@@ -1443,11 +1433,13 @@ def setUp(self):
         if len(self.hosts) < 2:
             self.skipTest("Requires at least two hosts for performing migration related tests")
 
+        for host in self.hosts:
+            print(f"=== setup() === Host.OS = {host.details['Host.OS']}")
+            if host.details['Host.OS'] in ['CentOS']:
+                self.skipTest("live migration is not stabily supported on CentOS")
+
```

Comment:
@DaanHoogland can we have this in a separate PR to get this merged earlier to fix failing migration tests on Trillian https://github.com/apache/cloudstack/issues/4757?
Assuming Marvin3 work might need some more time.
cc @rhtyd  @weizhouapache

Reply:
will do

Reply:
see #4801